style.css

```diff
@@ -0,0 +1,83 @@
+*{
+  margin: 0;
+  padding: 0;
+}
+body{
+  background-color: #121212;
+  font-family: montserrat;
+}
+
+.sidebar{
```

Comment:
@Bishakha1589 These 4 lines of css do nothing. These are the default css and thus, not required. Please correct this.